src/components/views/homepage/HomeChangeMakers.tsx

```diff
@@ -64,12 +64,14 @@ const HomeChangeMakers = () => {
 			<PurpleArc />
 			<CyanArc />
 			<MustardArc />
-			<D3>Calling all Changemakers!</D3>
-			<br />
-			<Lead>
-				Do you have a for-good project that&apos;s creating value for
-				society, for the environment, or for the world?
-			</Lead>
+			<TitleContainer>
```

Comment:
why did you add this?

Reply:
Sorry, I forgot to remove, I was just testing some stuff.